src/main/java/org/folio/rest/impl/PutOrdersByIdHelper.java

```diff
@@ -0,0 +1,113 @@
+package org.folio.rest.impl;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import javax.ws.rs.core.Response;
+
+import org.apache.log4j.Logger;
+import org.folio.orders.rest.exceptions.HttpException;
+import org.folio.rest.jaxrs.model.CompositePurchaseOrder;
+import org.folio.rest.jaxrs.resource.OrdersResource.PutOrdersByIdResponse;
+import org.folio.rest.tools.client.interfaces.HttpClientInterface;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.json.JsonObject;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+public class PutOrdersByIdHelper {
+  private static final Logger logger = Logger.getLogger(DeleteOrdersByIdHelper.class);
```

Comment:
For consistency probably want to use the vertx logger like the other helpers were recently switched to... e.g. `LoggerFactory.getLogger(...)` see: 
https://github.com/folio-org/mod-orders/blob/2d80a99b2696b6a608cc84bfe9d6f57e9dbe614c/src/main/java/org/folio/rest/impl/PostOrdersHelper.java#L30
and
https://github.com/folio-org/mod-orders/blob/2d80a99b2696b6a608cc84bfe9d6f57e9dbe614c/src/main/java/org/folio/rest/impl/PostOrdersHelper.java#L36